src/test/jolie/test.ol

```diff
@@ -1,2 +1,36 @@
-/** TODO: Test for the db-connector here */
+include "db_connector_iface.iol"
```

Comment:
We can just include "../../main/jolie/db_connector.iol" (I hope I got the path right :-) )